jenkins-slaves/jenkins-slave-golang/go.mod

```diff
@@ -0,0 +1,8 @@
+module example.com/hello
```

Comment:
since go bits added are for a test app. should they live under an "example" or "test" dir? just thinking not to confuse people as to why there are some go files in the root dir.

@pabrahamsson ; thoughts?

Reply:
I agree with you. Should I go ahead and make this change @pabrahamsson ?